ee/tabby-webserver/src/service/background_job/third_party_integration.rs

```diff
@@ -179,11 +184,16 @@ impl SchedulerGithubGitlabJob {
     }
 }
 
+pub struct FetchState {
+    updated_at: DateTime<Utc>,
+    should_clean: bool,
+}
```

Comment:
```suggestion
pub StructuredDocState  {
    updated_at: DateTime<Utc>,
    deleted: bool,
}
```

Move definition to `tabby-index/src/structured_doc/public.rs`